custom_components/bosch_alarm/services.yaml

```diff
@@ -0,0 +1,12 @@
+set_date_time:
+  target:
+    entity:
+      domain: alarm_control_panel
```

Comment:
Looking at what some other integrations are doing, I think we should also add:
```
integration: bosch_alarm
```

Otherwise, LGTM, so after that please feel free to squash and merge.